packages/amazonq/test/unit/codewhisperer/tracker/userWrittenCodeTracker.test.ts

```diff
@@ -0,0 +1,194 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import * as sinon from 'sinon'
+import * as vscode from 'vscode'
+import { UserWrittenCodeTracker, TelemetryHelper, AuthUtil } from 'aws-core-vscode/codewhisperer'
+import { createMockDocument, resetCodeWhispererGlobalVariables } from 'aws-core-vscode/test'
+
+describe('userWrittenCodeTracker', function () {
+    describe('test isActive', function () {
```

Comment:
tests never need "test" in their name
```suggestion
    describe('isActive()', function () {
```